src/world/Server/Packets/Handlers/ChatHandler.cpp

```diff
@@ -121,33 +121,33 @@ void WorldSession::handleMessageChatOpcode(WorldPacket& recvPacket)
     if (!srlPacket.deserialise(recvPacket))
         return;
 
-    auto language = srlPacket.language;
```

Comment:
it is possible to use more detailed 😉
_packet_language
_language
_chat_language

Reply:
You're right. I will update it soon.